frontend/packages/kubevirt-plugin/src/components/vms/ActionMessage.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react';
+import { Stack, StackItem } from '@patternfly/react-core';
+import { Trans, useTranslation } from 'react-i18next';
+import { K8sResourceCommon } from '@console/internal/module/k8s/types';
+import { getName, getNamespace } from '@console/shared';
+import { RestartDiskAlert } from '../vm-disks/RestartDiskAlert';
+
+export const ActionMessage: React.FC<{
```

Comment:
is this a `ActionMessage` for all actions or a disk specific action message ?

Reply:
basically it applies only for stopping and restarting VMs at the moment